blob/blob.go

```diff
@@ -41,6 +45,14 @@ func (b *blobImpl) Name() string {
 	return b.name
 }
 
+func (b *blobImpl) Seek(sectorSize uint16) {
```

Comment:
Remove `at` and make `Seek` conform to the `io.Seeker` interface.

Reply:
Done.